src/interface/src/app/map/map.component.html

```diff
@@ -46,83 +37,59 @@ <h3>Legend</h3>
-              (change)="toggleHuc12BoundariesLayer(map)">
+            <mat-checkbox name="{{ map.id + '-huc12-toggle' }}" aria-label="Select or deselect" class="layer-checkbox"
+              [(ngModel)]="map.config.showHuc12BoundaryLayer" (change)="toggleHuc12BoundariesLayer(map)">
               HUC-12
             </mat-checkbox>
             <mat-spinner [diameter]="24" *ngIf="!huc12BoundaryGeoJsonLoaded"></mat-spinner>
           </div>
           <div class="layer-control-container">
-            <mat-checkbox
-              name="{{ map.id + '-huc10-toggle' }}"
-              aria-label="Select or deselect"
-              class="layer-checkbox"
-              [(ngModel)]="map.config.showHuc10BoundaryLayer"
-              (change)="toggleHUC10BoundariesLayer(map)">
+            <mat-checkbox name="{{ map.id + '-huc10-toggle' }}" aria-label="Select or deselect" class="layer-checkbox"
+              [(ngModel)]="map.config.showHuc10BoundaryLayer" (change)="toggleHUC10BoundariesLayer(map)">
               HUC-10
             </mat-checkbox>
             <mat-spinner [diameter]="24" *ngIf="!huc10BoundaryGeoJsonLoaded"></mat-spinner>
           </div>
           <div class="layer-control-container">
-            <mat-checkbox
-              name="{{ map.id + '-county-toggle' }}"
-              aria-label="Select or deselect"
-              class="layer-checkbox"
-              [(ngModel)]="map.config.showCountyBoundaryLayer"
-              (change)="toggleCountyBoundariesLayer(map)">
+            <mat-checkbox name="{{ map.id + '-county-toggle' }}" aria-label="Select or deselect" class="layer-checkbox"
+              [(ngModel)]="map.config.showCountyBoundaryLayer" (change)="toggleCountyBoundariesLayer(map)">
               Counties
             </mat-checkbox>
             <mat-spinner [diameter]="24" *ngIf="!countyBoundaryGeoJsonLoaded"></mat-spinner>
           </div>
           <div class="layer-control-container">
-            <mat-checkbox
-              name="{{ map.id + '-usfs-toggle' }}"
-              aria-label="Select or deselect"
-              class="layer-checkbox"
-              [(ngModel)]="map.config.showUsForestBoundaryLayer"
-              (change)="toggleUSForestsBoundariesLayer(map)">
+            <mat-checkbox name="{{ map.id + '-usfs-toggle' }}" aria-label="Select or deselect" class="layer-checkbox"
+              [(ngModel)]="map.config.showUsForestBoundaryLayer" (change)="toggleUSForestsBoundariesLayer(map)">
               US Forests
             </mat-checkbox>
             <mat-spinner [diameter]="24" *ngIf="!usForestBoundaryGeoJsonLoaded"></mat-spinner>
           </div>
           <h3>Projects</h3>
           <div class="layer-control-container">
-            <mat-checkbox
-              name="{{ map.id + '-existing-projects-toggle' }}"
-              aria-label="Select or deselect"
-              class="layer-checkbox"
-              [(ngModel)]="map.config.showExistingProjectsLayer"
```

Comment:
Were these lines meant to be reformatted?

Reply:
I used autoformat on the file since I was already making some changes. Lemme know if the autoformatter did something weird and you want me to revert this in a follow up.